rust/src/dcerpc/dcerpc.rs

```diff
@@ -108,6 +109,10 @@ pub const DCERPC_TYPE_ORPHANED: u8 = 19;
 pub const DCERPC_TYPE_RTS: u8 = 20;
 pub const DCERPC_TYPE_UNKNOWN: u8 = 99;
 
+pub const PROTO_DCERPC: u16 = 9;
```

Comment:
I see. I want to understand why though. Its a constant value that never changes.
ALPROTO_* on the Rust side are static muts as mentioned in the issue and we assign the value of proto to them after registration so I am finding it difficult to use the proto number (in this case `9`) in the functions before registration happens.
Could you please tell what could be done if this is incorrect?
(e.g. I need to use this constant in function ` register_pattern_probe` which is called before the static mut `ALPROTO_DCERPC` is assigned)

Reply:
I guess the ALPROTO_* definitions should be moved to rust and then imported in C to solve https://redmine.openinfosecfoundation.org/issues/3524

Otherwise, maybe you should call register_pattern_probe after ALPROTO_DCERPC is assigned

Reply:
OK. I moved the pattern probe function after ALPROTO_DCERPC is assigned for now. I do not know if there can be any consequences though as it required me to add an unsafe block everywhere I used this variable (as its a static mut)